advisories/0010.md

```diff
@@ -0,0 +1,111 @@
+## MCR: runc process.cwd and leaked fds container breakout 
+### Release Date
+2024-01-31
+### Overview
+Several container breakouts due to internally leaked fds.
+
+Consult the [upstream advisory][upstream] and [snyk][snyk.io] for details, mitigations, workarounds, and patches.
+### Affected Products
+Mirantis Container Runtime (MCR) <= 23.0.9
+## Vulnerability Information
+### CVE Identifier
+[CVE-2024-21626][nvd.nist.gov]
+### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+### CWEs
+CWE-403, CWE-668
+### How to check if your environment is vulnerable
+`runc` version is equal or lower (<=) than `1.1.11-rc1`
+````
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1 <<<<<<<<<<<<<<<
+````
+## How to fix
+### MCR 23.0.x
+Upgrade MCR packages to the fixed 23.0.9-1 version or newer
+* `docker-ee*` packages to:
+  - `23.0.9~4-0` Ubuntu
+  - `23.0.9-4` for RPM package based OSes
+* `containerd.io` package to
+  - `1.6.28~rc.1-2` Ubuntu
+  - `1.6.28-4.1.rc.1.1` for RPM package based OSes
+* `Note:` Restart of the `dockerd` service is `NOT` necessary! 
+#### Ubuntu
+```
+# apt-get install docker-ee=5:23.0.9~4-0* docker-ee-cli=5:23.0.9~4-0* docker-ee-rootless-extras=5:23.0.9~4-0* containerd.io=1.6.28~rc.1-2*
```

Comment:
```suggestion
# apt-get install 'docker-ee=5:23.0.9~4-0*' 'docker-ee-cli=5:23.0.9~4-0*' 'docker-ee-rootless-extras=5:23.0.9~4-0*' 'containerd.io=1.6.28~rc.1-2*'
```
We need to tell them something so the asterisks don't get shell-expanded. If you don't like the single quotes, backslash-escape them.